addons/binding/org.openhab.binding.cm11a/src/main/java/org/openhab/binding/cm11a/handler/Cm11aBridgeHandler.java

```diff
@@ -184,27 +184,36 @@ private ChannelUID findX10Channel(List<Channel> channels) {
      * @param dims
      */
     private void updateX10State(Thing thing, ChannelUID channelUid, X10COMMAND cmd, int dims) {
+        if (thing == null) {
```

Comment:
Should this maybe be handled in the caller of this method?

Reply:
Appears to already be handled in the caller. Handling it here prevents this method from failing if a new caller appears later that sends in a null. But I'll remove this if you prefer.

Reply:
I have mixed feelings about it, if we would start checking for everything that can become `null` later in the lifecycle of an application we would not be able to see the code in between the `null`-checks.